docs/dev.rst

```diff
@@ -1,20 +1,522 @@
-Develop a project
+Develop a Project
 =================
 
-Set up a dev org
+A general overview on how to develop a Salesforce project with CumulusCI.
+
+
+
+Set Up a Dev Org
 ----------------
 
-Track and retrieve changes
---------------------------
+The ``dev_org`` flow creates an org to develop on by moving all metadata (managed and unmanaged) into the org, and configuring it to be ready for development.
```

Comment:
To confirm: "To develop on" is the correct way to discuss working with orgs?

Reply:
I think both "on" and "in" are suitable here.